doc/source/index.rst

```diff
@@ -0,0 +1,10 @@
+.. include:: ../../README.rst
+
```

Comment:
An introductory sentence should be added here.

Reply:
The introduction here is the one in the `README.md` file, as this file gets included directly into the documentation.

Nevertheless, I think the introduction in the `README.md` file can be improved.